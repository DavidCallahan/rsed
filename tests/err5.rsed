print "foo"
dsfsdf 
if "bar" then
   sdfsf
else
    print "ok"
end
print  sdfdf " foo "
print "foo"