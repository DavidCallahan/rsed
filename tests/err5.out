3; syntax error, unexpected NEWLINE, expecting '='
5; syntax error, unexpected NEWLINE, expecting '='
8; syntax error, unexpected STRING, expecting '('
